### PR TITLE
S2t1 form

### PR DESCRIPTION
**Pull request by AndrewScott85**

### CODE REVIEW

File: addSandwich.php [hunk 1]
```diff
@@ -0,0 +1,146 @@
+<?php
+require_once 'db.php';
+//echo '<pre>';
+//print_r($_POST);
+//echo '</pre>';
```
Comment (patrick-holden) [inline addSandwich.php]: remove commented code

File: css/styles.css [hunk 2]
```diff
@@ -1,30 +1,60 @@
...
 }
 
-.content {
+.image {
+    width: 100%;
+    height: auto;
+    border-top: solid 1px #000000;
+    border-bottom: solid 1px #000000;
+    background-color: gray;
+}
```
Comment (patrick-holden) [inline css/styles.css:24]: insert empty line

File: css/styles.css [hunk 3]
```diff
@@ -1,30 +1,60 @@
...
+    transform: scale(1.05);
+}
+
+form {
+    display: flex;
+    flex-direction: column;
+    width: 50vw;
+}
+
+@media only screen and (min-width: 700px) {
```
Comment (patrick-holden) [inline css/styles.css:46]: delete empty line

**Review by patrick-holden — CHANGES_REQUESTED** [review]

File: css/styles.css (hunk 3 above)
Comment (AndrewScott85) [reply css/styles.css:46]: no

**Review by patrick-holden — APPROVED** [review]

**Pull request merged**